.github/workflows/kind-e2e.yml

```diff
@@ -3,9 +3,6 @@ on: [push, pull_request]
 jobs:
   e2e-kind:
     runs-on: ubuntu-latest
-    if: >
-      (( github.event.pull_request.head.repo.owner.login != github.event.pull_request.base.repo.owner.login ) &&
-        github.event_name == 'pull_request' ) || (github.event_name == 'push' && github.event.commits != '[]' )
```

Comment:
As far as I add in the comment, these conditional runs e2e-kind test, so keep this.

Reply:
I see no reason not to run these tests in PRs.
Consider the scenario where a change breaks a feature and an E2E test starts failing. There is the risk of discovering that failure only after merging that change into master, and it can produce another change that can break something else.

Please, consider this scenario.